app/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 
     defaultConfig {
         applicationId "me.hackerchick.catima"
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
Hmm, this kinda sucks. Means having to leave 35 users without updates. I agree that we don't seem to have another choice though. Just leaving this comment here to let you know that if you find some clever trick to stop us from needing to drop Android 4.4 support that would be cool.